docs/src/components/productBaseUI/components/BaseButtonDemo.tsx

```diff
@@ -5,37 +5,47 @@ import { styled, GlobalStyles } from '@mui/system';
 
 const buttonStyles = `
   font-family: 'IBM Plex Sans', sans-serif;
-  font-weight: bold;
+  font-weight: 600;
   font-size: 0.875rem;
-  background-color: var(--palette-primary);
-  padding: 10px 16px;
+  line-height: 1.5;
+  background: var(--primary);
+  padding: 6px 12px;
   border-radius: 8px;
   color: white;
   transition: all 150ms ease;
   cursor: pointer;
   border: none;
-  box-shadow: var(--shadow);
+  box-shadow: var(--solid-btn-shadow);
+  user-select: none;
 
   &:hover {
-    background-color: var(--palette-primary-hover);
+    background: var(--primary-hover);
+    box-shadow: none;
   }
 
   &.Mui-active {
-    background-color: var(--palette-primary-dark);
+    background: var(--primary-active);
   }
 
   &.Mui-focusVisible {
-    outline: 4px solid var(--focus-ring);
+    outline: 3px solid var(--focus-ring);
+    outline-offset: 2px;
   }
 
   &.Mui-disabled {
     opacity: 0.5;
     cursor: not-allowed;
+
```

Comment:
```suggestion
'box-shadow: none;' 
```

The new styles with box-shadows are causing the hover on disabled button to trigger the 'box-shadow: none;' effect, making it have the same animation as the enabled button. 

Maybe we could remove the shadows from the disabled button

Reply:
Good catch!